packages/aggregator/src/Aggregator.js

```diff
@@ -32,9 +32,14 @@ export class Aggregator {
     return this.dataBroker.getDataSourceTypes();
   }
 
-  processAnalytics = (analytics, aggregationOptions, requestedPeriod) => {
+  processAnalytics = (analytics, aggregationOptions, aggregationsProcessed, requestedPeriod) => {
```

Comment:
Thoughts on the term `aggregationsProcessed`? Feels a bit wordy to me, but couldn't really think of anything nicer...